conans/client/generators/cmake_common.py

```diff
@@ -642,6 +642,8 @@ class CMakeCommonMacros:
 
             link_directories(${CONAN_LIB_DIRS})
 
+            conan_find_libraries_abs_path("${CONAN_LIBS}" "${CONAN_LIB_DIRS}"
```

Comment:
I am very concerned about this change. From experience I'd say that this will probably break users that are checking, iterating or doing something with the CONAN_LIBS variable.

Reply:
Yes, I think that's right, It's probably not worth the risk, I did not take into account that maybe users are checking the current CONAN_LIBS variable and we could break them.